src/rogue/interfaces/memory/TcpClient.cpp

```diff
@@ -110,10 +110,15 @@ rim::TcpClient::TcpClient (std::string addr, uint16_t port) : rim::Slave(4,0xFFF
 
 //! Destructor
 rim::TcpClient::~TcpClient() {
-  this->close();
+  this->stop();
 }
 
+// deprecated
 void rim::TcpClient::close() {
+   this->stop();
```

Comment:
Need to add a print statement about how  `close()` is deprecated and will be removed in the future and should use `stop()` instead